### PR TITLE
Add common proto files to Python grpcio-tools

### LINKED ISSUE #6738: Include .proto files for common types in python's grpcio-tools package
The grpcio-tools package does not currently include common .proto files. These are needed when generating code for .proto files that import common .protos.
Protobuf's binary package include these files, so we should do the same.

### PR DESCRIPTION
**Pull request by soltanmm**
Fixes #6738.

~~Build to follow before being merge-ready: https://grpc-testing.appspot.com/view/Artifacts/job/gRPC_build_artifacts/983/ .~~

Build to follow before being merge-ready: https://grpc-testing.appspot.com/view/Artifacts/job/gRPC_build_artifacts/985/ .

### CODE REVIEW

File: tools/distrib/python/grpcio_tools/protoc_lib_deps.py [hunk 1]
```diff
@@ -30,3 +30,7 @@
...
+PROTO_FILES=['google/protobuf/wrappers.proto', 'google/protobuf/type.proto', 'google/protobuf/timestamp.proto', 'google/protobuf/struct.proto', 'google/protobuf/source_context.proto', 'google/protobuf/field_mask.proto', 'google/protobuf/empty.proto', 'google/protobuf/duration.proto', 'google/protobuf/descriptor.proto', 'google/protobuf/compiler/plugin.proto', 'google/protobuf/api.proto', 'google/protobuf/any.proto']
```
Comment (nathanielmanistaatgoogle) [inline tools/distrib/python/grpcio_tools/protoc_lib_deps.py]: Does this have to be all one line?

Comment (nathanielmanistaatgoogle) [inline tools/distrib/python/grpcio_tools/protoc_lib_deps.py]: Whoops; it's generated, isn't it?

**Comment by nathanielmanistaatgoogle** [comment]
How do the build results look? What's up with the Python x86 MacOS cell?

File: tools/distrib/python/grpcio_tools/protoc_lib_deps.py (hunk 1 above)
Comment (soltanmm) [inline tools/distrib/python/grpcio_tools/protoc_lib_deps.py]: Jes.

**Comment by soltanmm** [comment]
We don't build for x86 Mac OS X.

**Comment by nathanielmanistaatgoogle** [comment]
Luhguhtum; should @kpayson64 also review or is this ready for merge?

**Comment by kpayson64** [comment]
lgtm

**Pull request merged**